### DISCUSSION TITLE
qBittorrent LXC update

### DISCUSSION THREAD

**Discussion by iz8mbw** [tteck/Proxmox#2582] [Q&A] [answered]
Hello,
just a question.
The currently version of qBittorrent inside the qBittorrent LXC is almost old (year 2022).
Is there a plan to update the qBittorrent LXC or a way to update qBittorrent inside the LXC?

Thanks.

**Comment by tteck** [comment, marked as answer]
The script installs the [Debian Bookworm package](https://packages.debian.org/stable/net/qbittorrent-nox) and updates via `apt update && apt upgrade`
https://github.com/tteck/Proxmox/blob/e5ec531d4f1a383ef7d4f9468457e0bb3a08e096/install/qbittorrent-install.sh#L23

**Reply by tteck** [reply]
The Debian [unstable development branch](https://packages.debian.org/sid/qbittorrent-nox) has a newer version.

**Comment by iz8mbw** [comment]
So, basically, the LXC is going to use the version of the official Debian repository.

**Reply by tteck** [reply]
Correct.

**Reply by chenks** [reply]
so how do we get the official Debian repository updated? as 4.6.5 is the current public release and has been out for some time.
running an old version without the option to get the most recent version might cause issues.

**Reply by tesla-76** [reply]
I followed these steps to upgrade to the latest release.
Edit the file  `/etc/apt/sources.list`  of the qBittorrent container by adding the string as the first line:
`deb http://ftp.debian.org/debian sid main contrib`

Restart the LXC container, then from the LXC console give the commands:
`sudo apt-get update`
`sudo apt-get full-upgrade`
Confirm each update or option, when finished restart the LXC container.

**Reply by chenks** [reply]
> I followed these steps to upgrade to the latest release. Edit the file `/etc/apt/sources.list` of the qBittorrent container by adding the string as the first line: `deb http://ftp.debian.org/debian sid main contrib`
> 
> Restart the LXC container, then from the LXC console give the commands: `sudo apt-get update` `sudo apt-get full-upgrade` Confirm each update or option, when finished restart the LXC container.

what impact does this have other than just qbittorrent?

**Comment by denes44** [comment]
An option to install the latest version from source would be nice.
